### PR TITLE
add doc for sysvar `tidb_session_alias`

### PR DESCRIPTION
**Pull request by lcwangchao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14972
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -4383,6 +4383,13 @@ SHOW WARNINGS;
 - Range: `[128, 9223372036854775807]`, in bytes. The memory format with the units "KB|MB|GB|TB" is also supported.
 - After you enable the memory limit, TiDB will terminate the SQL statement with the highest memory usage on the current instance. This variable specifies the minimum memory usage of the SQL statement to be terminated. If the memory usage of a TiDB instance that exceeds the limit is caused by too many sessions with low memory usage, you can properly lower the value of this variable to allow more sessions to be canceled.
 
+### `tidb_session_alias` <span class="version-mark">New in v7.4.0</span>
+
+- Scope: SESSION
+- Persists to cluster: No
+- Default value: ""
+- You can use this variable to customize the value of the `session_alias` column in the logs related to the current session. This setting affects the logs of multiple nodes during the execution of the statement (including TiKV). The maximum length of this variable is limited to 64 characters, and any exceeding length will be truncated automatically. Spaces at the end of the variable value will also be removed automatically.
```
Comment (songrijie) [inline system-variables.md:4391]:
```suggestion
- You can use this variable to customize the value of the `session_alias` column in the logs related to the current session. This setting affects the logs of multiple nodes during the execution of the statement (including TiKV). The maximum length of this variable is limited to 64 characters, and any exceeding length will be truncated automatically. Spaces at the end of the value will also be removed automatically.
```

**Review by songrijie — COMMENTED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -4383,6 +4383,13 @@ SHOW WARNINGS;
 - Range: `[128, 9223372036854775807]`, in bytes. The memory format with the units "KB|MB|GB|TB" is also supported.
 - After you enable the memory limit, TiDB will terminate the SQL statement with the highest memory usage on the current instance. This variable specifies the minimum memory usage of the SQL statement to be terminated. If the memory usage of a TiDB instance that exceeds the limit is caused by too many sessions with low memory usage, you can properly lower the value of this variable to allow more sessions to be canceled.
 
+### `tidb_session_alias` <span class="version-mark">New in v7.4.0</span>
+
+- Scope: SESSION
+- Persists to cluster: No
+- Default value: ""
+- You can use this variable to customize the value of the `session_alias` column in the logs related to the current session. This setting affects the logs of multiple nodes during the execution of the statement (including TiKV). The maximum length of this variable is limited to 64 characters, and any exceeding length will be truncated automatically. Spaces at the end of the value will also be removed automatically.
```
Comment (qiancai) [inline system-variables.md:4391]:
```suggestion
- You can use this variable to customize the value of the `session_alias` column in the logs related to the current session, which helps identify the session in troubleshooting. This setting affects the logs of multiple nodes involved in the statement execution (including TiKV). The maximum length of this variable is limited to 64 characters, and any character exceeding the length will be truncated automatically. Spaces at the end of the value will also be removed automatically.
```

Comment (qiancai) [reply system-variables.md:4391]: multiple statements might be executed in one session, so  "the execution of the statement" might be inaccurate

File: system-variables.md [hunk 3]
```diff
@@ -4383,6 +4383,13 @@ SHOW WARNINGS;
...
+
 ### `tidb_session_plan_cache_size` <span class="version-mark">New in v7.1.0</span>
```
Comment (hfxsd) [inline system-variables.md:4393]:
```suggestion
### tidb_session_plan_cache_size <span class="version-mark">New in v7.1.0</span>
```

File: system-variables.md [hunk 4]
```diff
@@ -4383,7 +4383,14 @@ SHOW WARNINGS;
 - Range: `[128, 9223372036854775807]`, in bytes. The memory format with the units "KB|MB|GB|TB" is also supported.
 - After you enable the memory limit, TiDB will terminate the SQL statement with the highest memory usage on the current instance. This variable specifies the minimum memory usage of the SQL statement to be terminated. If the memory usage of a TiDB instance that exceeds the limit is caused by too many sessions with low memory usage, you can properly lower the value of this variable to allow more sessions to be canceled.
 
-### `tidb_session_plan_cache_size` <span class="version-mark">New in v7.1.0</span>
+### tidb_session_alias <span class="version-mark">New in v7.4.0</span>
+
+- Scope: SESSION
+- Persists to cluster: No
```
Comment (Oreoxmt) [inline system-variables.md:4389]:
```suggestion
- Persists to cluster: No
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
```

**Pull request merged**